webapp/spec.py

```diff
@@ -0,0 +1,120 @@
+import io
+
+from apiclient.http import MediaIoBaseDownload
+from bs4 import BeautifulSoup
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+
+from dateutil.parser import parse
+
+from flask import abort
+
+from settings import SERVICE_ACCOUNT_INFO
+
+
+class GoogleDrive:
+    def __init__(self):
+        scopes = [
+            "https://www.googleapis.com/auth/drive.readonly",
+        ]
+        service_account_info = SERVICE_ACCOUNT_INFO
+        credentials = service_account.Credentials.from_service_account_info(
+            service_account_info, scopes=scopes
```

Comment:
No need to redefine this variable? 
```suggestion
        credentials = service_account.Credentials.from_service_account_info(
            SERVICE_ACCOUNT_INFO, scopes=scopes
```